helpers/components/addOrderedIds.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const name = getName(__filename),
+  filter = require('./' + name);
+
+describe(name, function () {
+  const content = [{
+    _ref: 'localhost/components/fake/instances/1',
+    text: 'hello'
+  },{
+    _ref: 'localhost/components/fake/instances/2',
+    text: 'hola'
+  },{
+    _ref: 'localhost/components/fake/instances/3',
+    text: 'bonjour'
+  }];
+
+  it('adds an orderedId property to all components in the list', function () {
+    expect(filter(content, 'mock-prefix-')).to.deep.equal([{
```

Comment:
IMO, we should be testing by compiling and running data through actual handlebars templates, as we do in our other helper tests, rather than testing the helper function directly. For example, in this one you'd write `const tpl = hbs.compile('{{#each (addOrderedIds content "mock-prefix-"}}')}}{{orderedId}}{{/each}}`, and check with something like `expect(tpl({content: [{},{},{}]}).to.equal('mock-prefix-1mock-prefix-2mock-prefix-3')`